assets/section-main-product.css

```diff
@@ -11,13 +11,14 @@
   margin: 0 auto;
 }
 
-.product__media-wrapper {
-  padding-left: 0;
+@media screen and (min-width: 750px) {
+  .product--reverse {
+    flex-direction: row-reverse;
```

Comment:
Have we validated with accessibility folks if there's any issue with this approach as it changes the tab order? I tested and it seems to works fine, but just wanted to know if there's any conversation I'm missing.

I like that we have `Skip to product information` always there so it's never an issue to tab through items :)

Reply:
That's a good point. I spoke to Scott about this and confirmed that we shouldn't be using `flex-direction: row-reverse;` for this type of layout option change in order to maintain our accessibility standards + have the correct DOM / tab order.

**TL;DR**
The "Skip to product information" link will remain when using the default `left` media position and will not be shown when using the `right` layout.